extensions/cl_mobileye_reservation_sets.asciidoc

```diff
@@ -0,0 +1,545 @@
+include::{generated}/api/version-notes/clCreateReservationSetMOBILEYE.asciidoc[]
+
+  * _command_buffer_ is a valid command buffer object.
+  * _affinity_domain_ is the level of the cache hierarchy that the work-groups
+    in the reservation-set with the same ID are guaranteed to share.For example,
+    if _affinity_domain_ is `CL_DEVICE_AFFINITY_DOMAIN_L2_CACHE`, and there are
+    two ND-range kernel commands in the reservation-set, work-groups with ID 0
+    from the two commands will execute on the same L2 data cache.
+    The value of _affinity_domain_ must be a valid value of
+    `cl_device_affinity_domain`.
+  * _num_sync_points_in_wait_list_ is the number of synchronization points in
+    the wait list.
+  * _sync_point_wait_list_ is a list of synchronization points that the
+    reservation-set must wait for before it can be used. It can be `NULL` if
+    _num_sync_points_in_wait_list_ is 0.
+  * _errcode_ret_ is an optional pointer to a variable that will receive the
+    error code if the function fails. If it is `NULL`, no error code will be
+    returned.
+
+*clCreateReservationSetMOBILEYE* returns a valid non-zero reservation-set and
+_errcode_ret_ is set to `CL_SUCCESS` if the reservation-set is created
+successfully. Otherwise, it returns a `NULL` value with one of the following
+error values returned in _errcode_ret_:
+
+  * `CL_INVALID_COMMAND_BUFFER_KHR` if _command_buffer_ is not a valid command
+    -buffer.
+  * `CL_INVALID_VALUE` if _affinity_domain_ is not a valid value of
+    `cl_device_affinity_domain`.
+  * `CL_INVALID_OPERATION` if _command_buffer_ has been finalized.
+  * `CL_INVALID_SYNC_POINT_WAIT_LIST_KHR` if _sync_point_wait_list_ is not
+    `NULL` and _num_sync_points_in_wait_list_ is 0, or if _sync_point_wait_list_
+    is `NULL` and _num_sync_points_in_wait_list_ is greater than 0, or if the
+    synchronization-point objects in _sync_point_wait_list_ are not valid
+    synchronization-points.
+  * `CL_OUT_OF_RESOURCES` if there is a failure to allocate resources
+    required by the OpenCL implementation on the device.
+  * `CL_OUT_OF_HOST_MEMORY` if there is a failure to allocate resources
+    required by the OpenCL implementation on the host.
+--
+
+[open,refpage='clRetainReservationSetMOBILEYE',desc='Increments the reservation-set object reference count.',type='protos']
+--
+To retain a reservation-set object, call the function
+
+include::{generated}/api/protos/clRetainReservationSetMOBILEYE.txt[]
+include::{generated}/api/version-notes/clRetainReservationSetMOBILEYE.asciidoc[]
+
+  * _reservation_set_ specifies the reservation-set object to be retained.
+
+The _reservation_set_ reference count is incremented.
+
+*clRetainReservationSetMOBILEYE* returns `CL_SUCCESS` if the function is
+executed successfully.
+Otherwise, it returns one of the following errors:
+
+  * `CL_INVALID_COMMAND_BUFFER_KHR` if _command_buffer_ is not a valid command
+    -buffer.
```

Comment:
Think it's a bit confusing using `CL_INVALID_COMMAND_BUFFER_KHR` for the error code with reservation step retain/release. The more generic `CL_INVALID_VALUE` might be better. 

Also here "not a valid command-buffer" -> "not a valid reservation-set object"